css/src/components/sidesheet.css

```diff
@@ -99,3 +118,11 @@
 .Sidesheet-header--shiftLeft {
   animation: shift-left-spacing-3 80ms cubic-bezier(0.2, 0, 0.38, 0.9);
 }
+
+.Sidesheet-body--nextPage {
+  animation: next-page-opacity 120ms cubic-bezier(0.2, 0, 0.38, 0.9);
+}
+
+.Sidesheet-body--firstPage {
+  animation: first-page-opacity 120ms cubic-bezier(0.4, 0.14, 0.3, 1);
+}
```

Comment:
Please use the existing animation classes

Reply:
Replaced following : 
120s : var(--duration--fast-02)
cubic-bezier(0.2, 0, 0.38, 0.9): var(--standard-productive-curve)
cubic-bezier(0.4, 0.14, 0.3, 1) : var(--standard-expressive-curve)